astronomer/providers/amazon/aws/triggers/emr.py

```diff
@@ -71,6 +71,113 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
             return
 
 
+class EmrContainerOperatorTrigger(BaseTrigger):
```

Comment:
we can write a base trigger class which can be used by the container sensor and async container operator

Reply:
I'll do that in separate PR tracking here https://github.com/astronomer/astronomer-providers/issues/243